src/main/java/dev/streamx/sling/connector/selectors/content/ResourceContentRelatedResourcesSelectorConfig.java

```
@@ -40,7 +40,8 @@
   @AttributeDefinition(
       name = "References Search Regexes",
       description = "Array of regex patterns used to search for paths within "
-          + "the textual representation of a Resource.",
+          + "the textual representation of a Resource."
+          + "Every regex must contain the desired expression wrapped in parentheses",
       type = AttributeType.STRING
```

Analysis of code has shown that the regex must contain the expression in `(` `)` (for example: `(.*)`)

---

I think it should still be thrown.
We cannot handle it to use some default values for the regexes.
Let's fail fast, to make the user know there is an error in config that needs to be fixed